repo/packages/J/jenkins/10/config.json

```diff
@@ -63,6 +63,22 @@
                 }
             }
         },
+         "roles": {
+              "description": "Role configuration properties for Jenkins on DC/OS.",
+              "type": "object",
+              "properties": {
+                "jenkins-master-role": {
+                    "description": "The accepted resource roles (e.g. slave_public) that the Jenkins master itself runs as using Marathon. By default, this will deploy to any agents with the * role.",
```

Comment:
Let's move this example to the end, after the statement about default. Rephrase to something like `For example, to deploy to a public DC/OS agent, set this to 'slave_public'.`